observability/monitoring/common-monitoring-configuration-scenarios.adoc

```diff
@@ -0,0 +1,62 @@
+:_mod-docs-content-type: ASSEMBLY
+[id="common-monitoring-configuration-scenarios"]
+= Common monitoring configuration scenarios
+include::_attributes/common-attributes.adoc[]
+:context: common-monitoring-configuration-scenarios
+
+toc::[]
+
+After {product-title} is installed, core platform monitoring components immediately begins collecting metrics, which you can query and view.
+The default in-cluster monitoring stack includes the core platform Prometheus instance that collects metrics from your cluster and the core Alertmanager instance that routes alerts, among other components.
+Depending on who will use the monitoring stack and for what purposes, as a cluster administrator, you can further configure these monitoring components to suit the needs of different users in various scenarios.
+
+[NOTE]
+====
+In addition to core platform monitoring, you can also optionally xref:../monitoring/enabling-monitoring-for-user-defined-projects.adoc#enabling-monitoring-for-user-defined-projects[enable monitoring for user-defined projects] for user workload monitoring.
+After doing so, users can then monitor their own services and workloads without the need for an additional monitoring solution.
+====
+
+[id="configuring-core-platform-monitoring-first-steps_{context}"]
+== Configuring core platform monitoring: First steps
+
+After {product-title} is installed, cluster administrators typically perform certain basic activities to configure core platform monitoring to suit their needs.
+These activities include setting up storage amd configuring options for Prometheus, Alertmanager, and other monitoring components.
+
+[NOTE]
+====
+By default, in a newly installed {product-title} system, users can query and view collected metrics.
+You need only configure an alert receiver if you want users to receive alert notifications.
+Any other configuration options listed here are optional.
+====
+
+* xref:../../observability/monitoring/configuring-the-monitoring-stack.adoc#creating-cluster-monitoring-configmap_configuring-the-monitoring-stack[Create the `cluster-monitoring-config` `ConfigMap` object] if it does not exist.
+* xref:../../observability/monitoring/managing-alerts.adoc#sending-notifications-to-external-systems_managing-alerts[Configure alert receivers] so that Alertmanager can send alerts to an external notification system such as email, Slack, or PagerDuty.
+* For shorter term data retention, xref:../../observability/monitoring/configuring-the-monitoring-stack.adoc#configuring-persistent-storage_configuring-the-monitoring-stack[configure persistent storage] for Prometheus and Alertmanager to store metrics and alert data.
+Specify the metrics data retention parameters for Prometheus and Thanos Ruler.
++
+[NOTE]
+====
+By default, in a newly installed {product-title} system, the `ClusterOperator` resource reports a `PrometheusDataPersistenceNotConfigured` status message to remind you that storage is not configured.
```

Comment:
```suggestion
By default, in a newly installed {product-title} system, the monitoring `ClusterOperator` resource reports a `PrometheusDataPersistenceNotConfigured` status message to remind you that storage is not configured.
```
add `monitoring` explicitly

Reply:
I agree here. `ClusterOperator` is a term used by CVO and in its API definitions (see https://github.com/openshift/enhancements/blob/2d2b0250232a9b227f938d7d7172d04d78a681ee/dev-guide/cluster-version-operator/dev/clusteroperator.md?plain=1#L31). Using this here could be confusing.

We create a `ClusterOperator` object with the name `monitoring`. So +1 to Junqis suggestion.